### PR TITLE
Feature/crop cloud db hookup

### PR DESCRIPTION
**Pull request by peterger8y**
# Description

This pull request implements an external RDS PostgreSQL hookup, replacing the reliance on zipfiles
for the original crop_cloud.py file. 
Additionally, an identical of Jimmy Slagle's original file was created - 'crop_cloud_original.py' for future reference. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [] Complete, tested, ready to review and merge
- [x ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -147,3 +147,6 @@ cython_debug/
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
 /data/transcribed_stories/
+
+# data
+/data/story_images.zip
```
Comment (BritVandy) [inline .gitignore:152]: To make this pythonic, you need to add another line at the end.

File: app/utils/visualizations/crop_cloud.py [hunk 2]
```diff
@@ -181,51 +134,41 @@ def count_syllables(word):
...
     # scale the complexities so the sum is 1000
     # words['complexity'] = words['complexity'] / words['complexity'].sum()
     return words['complexity']
 
+
 # Compute the master scale needed to hit a certain density
 def get_scale(boxes, canvas_area, density):
     desired_area = canvas_area * density
-    complexity_word_area = sum(boxes.width * boxes.height * boxes.complexity**2)
+    complexity_word_area = sum(boxes.width * boxes.height * boxes.complexity ** 2)
```
Comment (BritVandy) [inline app/utils/visualizations/crop_cloud.py:149]: I like all the little changes that make it "prettier".

File: app/utils/visualizations/crop_cloud_original.py [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+# 2017-07-03 11:18:34
+# 2017-08-09 04:45:18
+# 2018-02-01 00:00:06
+# 2019-03-04 03:01:14
+# 2019-04-10 04:44:16
+# 2019-06-16 10:00:55
+# 2020-10-01 17:19:19
+# 2020-10-23 16:33:27
+# 2020-11-14 19:31:27
+
```
Comment (BritVandy) [inline app/utils/visualizations/crop_cloud_original.py:623]: Is there a better place for this documentation to go? Maybe a readme or a trello card?

**Review by BritVandy — APPROVED** [review]
Solid update to implement database access.

**Review by techthumb1 — APPROVED** [review]
Good job on the feature Peter, nice work! I believe that by creating a database, it causes the application to be much more efficient and secure than the previous code.

**Pull request merged**